go.mod

```diff
@@ -30,6 +30,12 @@ require (
 
 replace github.com/btcsuite/btcwallet/walletdb => ./walletdb
 
+replace github.com/btcsuite/btcd => github.com/guggero/btcd v0.20.1-beta.0.20220214155024-51fd177d32c0
+
+replace github.com/btcsuite/btcd/btcec/v2 => github.com/guggero/btcd/btcec/v2 v2.0.0-20220214155024-51fd177d32c0
+
+replace github.com/btcsuite/btcwallet/wallet/txauthor => ./wallet/txauthor
```

Comment:
FWIW this was removed in a prior PR, but I can see how it's helpful to fully integrate things in one swoop. Otherwise, we can make a new PR that adds the new `txscript` stuff, then does a tag, then to have it integrated here ultimately.